eng/build.ps1

```diff
@@ -59,6 +59,7 @@ param (
   [switch][Alias('test')]$testDesktop,
   [switch]$testCoreClr,
   [switch]$testIOperation,
+  [switch]$testUsedAssemblies,
```

Comment:
>$testUsedAssemblies [](start = 10, length = 19)

Should we pass this flag in one of the CI legs, or is this intended for local testing only at this point?

Reply:
> Should we pass this flag in one of the CI legs, or is this intended for local testing only at this point?

Once the feature is moved to master, I think.

---
In reply to: [367718165](https://github.com/dotnet/roslyn/pull/40987#discussion_r367718165) [](ancestors = 367718165)